Makefile

```diff
@@ -174,17 +177,17 @@ helm-chart-generate: kustomize helm kubectl-slice yq charts
 	rm -rf charts/$(CHART_NAME)/raw-files
 
 	# create and populate NOTES.txt
-	@echo "AWX Operator installed with Helm Chart version $(VERSION)" > charts/$(CHART_NAME)/templates/NOTES.txt
+	@echo "AWX Operator installed with Helm Chart version {{ .Chart.Version }}" > charts/$(CHART_NAME)/templates/NOTES.txt
```

Comment:
s/.Chart.Version/CHART_VERSION/g?

Reply:
NOTES.txt is templatable, so it will read it in live when the chart runs